Makefile

```diff
@@ -73,7 +73,7 @@ assemble-core-release:
 
 .PHONY: core-unit-tests
 core-unit-tests:
-	$(call run-gradle-tasks,$(CORE_MODULES),test)
```

Comment:
Jacoco is slow and heavy, could you add a separate makefile command for tests with coverage so that it doesn't impact the duration of tests that we run during local development?

Reply:
Interesting, checked the CI building times and sometimes they even take longer 

This build -> `9m 17s` / `2m 26s` https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/9084/workflows/e494e304-e6c5-4d10-bbd7-9a8f4a339e1d/jobs/33801

https://github.com/mapbox/mapbox-navigation-android/pull/3661 build -> `12m 45s` / `3m 4s` https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/9090/workflows/1b3e8626-1735-4867-a43b-30d123b7a072/jobs/33832 (not sure if we added a bunch of low tests as part of https://github.com/mapbox/mapbox-navigation-android/pull/3661)

https://github.com/mapbox/mapbox-navigation-android/pull/3664 build -> `9m 12s` / `2m 15s` https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/9085/workflows/38332f8f-217f-4988-bedb-6a55816a2f70/jobs/33808

Will add a separate command to scratch a couple of seconds 👍